vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -501,6 +505,16 @@ class LedgerImpl : public ledger::Ledger,
       ledger::Result result,
       std::unique_ptr<ledger::PublisherInfo> info);
 
+  void DownloadPublisherList(
+      ledger::LoadURLCallback callback);
+
+  void OnRefreshPublisher(
+      int response_status_code,
+      const std::string& response,
+      const std::map<std::string, std::string>& headers,
+      ledger::OnRefreshPublisherCallback callback,
```

Comment:
nit: can we do first publisher_key and then callback

Reply:
Done.